osf/migrations/0041_add_quickfiles.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-06-13 17:35
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations, models
+from django.core.management.sql import emit_post_migrate_signal
+
+from osf.models import OSFUser, QuickFiles
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def add_quickfiles(*args, **kwargs):
+    emit_post_migrate_signal(2, False, 'default')
```

Comment:
I thought this was necessary so that the models created earlier in the very same migration were available in the next function, as I had this problem in a previous PR where I was creating permissions and then wanted to assign them in groups in one migration -- found the [solution in this django ticket](https://code.djangoproject.com/ticket/23422) -- though I'm realizing that that might have been Permissions specifically created... so I'll remove it! (having trouble dropping my local DB completely, but reversing the migration seems to be OK without it...)